src/types/types.ts

```diff
@@ -667,6 +667,8 @@ export interface EdgeSpendInfo {
   noUnconfirmed?: boolean
   networkFeeOption?: 'high' | 'standard' | 'low' | 'custom'
   customNetworkFee?: JsonObject // Some kind of currency-specific JSON
+  /** Enables/disables RBF for currencies where RBF is opt-in or optional. */
+  enableRbf?: boolean
```

Comment:
This would read better as: "Enables RBF on chains where RBF is optional".

Also, you need to edit currency-wallet-api around line 460 to actually pass this new field down to the engine. We filter the object so unknown fields won't pass through.